internal/handlers/common/aggregations/number.go

```diff
@@ -19,13 +19,13 @@ import (
 	"math/big"
 )
 
-// sumNumbers accumulate numbers and returns the result of summation.
+// SumNumbers accumulate numbers and returns the result of summation.
```

Comment:
Shouldn't this function be in `operators`? Then only two places it's called from is in `sum.go` file.

Reply:
Yeah moved to operators.